config/passport.js

```diff
@@ -0,0 +1,57 @@
+const passport = require( 'passport' )
+const User = require( '../models/users' )
+const JwtStrategy = require( 'passport-jwt' ).Strategy
+const ExtractJwt = require( 'passport-jwt' ).ExtractJwt
+const LocalStrategy = require( 'passport-local' )
+
+const localOptions = {
+  usernameField: 'email'
+}
+
+const ERROR_MESSAGE = {
+  error: 'Your login details could not be verified. Please try again.'
+}
+
+const strategy = ( email, password, done ) => {
+
+  User.findOne({ email }, ( error, user ) => {
+    if( error ) {
+      return done( error )
+    }
+
+    if( ! user ) {
+      return done( null, false, ERROR_MESSAGE )
+    }
+
+    user.comparePassword(password, function( error, isMatch ) {
+      if ( error ) { return done(error); }
+      if ( !isMatch ) { return done(null, false, ERROR_MESSAGE ); }
+
+      return done( null, user );
+    })
+  })
+}
+
+const localLogin = new LocalStrategy( localOptions, strategy )
+
+const jwtOptions = {
+  jwtFromRequest: ExtractJwt.fromAuthHeader(),
+  secretOrKey: 'super secret passphrase'
```

Comment:
this secret needs to come from `process.env.SECRET`, and have a corresponding key set in the .env file.  See @Moniarchy and @jessehall3 work for how to accomplish this.